tests/php/Model/RedirectorPageTest.php

```diff
@@ -117,4 +117,9 @@ public function testNoJSLinksAllowed()
         $page->write();
         $this->assertEmpty($page->ExternalURL);
     }
+
+    public function testFileRedirector()
+    {
+        $this->assertContains("example.pdf", $this->objFromFixture(RedirectorPage::class, 'file')->Link());
```

Comment:
```suggestion
        $this->assertStringContainsString("example.pdf", $this->objFromFixture(RedirectorPage::class, 'file')->Link());
```
This syntax changed with the upgrade to PHPUnit 9.5 (https://docs.silverstripe.org/en/4/upgrading/phpunit9/)

Reply:
Done